ui/raidboss/data/05-shb/dungeon/twinning.js

```diff
@@ -224,24 +152,7 @@
       regexJa: Regexes.startsUsing({ id: '3DFB', source: 'タイクーン' }),
       regexCn: Regexes.startsUsing({ id: '3DFB', source: '泰空' }),
       regexKo: Regexes.startsUsing({ id: '3DFB', source: '타이쿤' }),
-      alertText: function(data, matches) {
-        if (data.me == matches.target) {
-          return {
-            en: 'Buster on YOU',
-            de: 'Buster auf DIR',
-            fr: 'Tankbuster sur VOUS',
-            ko: '나에게 탱버',
-          };
-        }
-        if (data.role == 'healer') {
-          return {
-            en: 'Buster on ' + data.ShortName(matches.target),
-            de: 'Buster auf ' + data.ShortName(matches.target),
-            fr: 'Tankbuster sur ' + data.ShortName(matches.target),
-            ko: '"' + data.ShortName(matches.target) + '"에게 탱버',
-          };
-        }
-      },
+      response: Responses.tankBuster(),
```

Comment:
Could you add a condition here of "target is you or role is healer" to preserve the previous logic?